imap/conversations.c

```diff
@@ -986,8 +994,8 @@ static void conv_to_buf(conversation_t *conv, struct buf *buf, int flagcount)
         nn = dlist_newlist(n, "THREAD");
         dlist_setguid(nn, "GUID", &thread->guid);
         dlist_setnum32(nn, "EXISTS", thread->exists);
-        dlist_setnum32(nn, "INTERNALDATE", thread->internaldate);
-        dlist_setnum32(nn, "CREATEDMODSEQ", thread->createdmodseq);
+        dlist_setnum64(nn, "INTERNALDATE", thread->internaldate);
```

Comment:
I'd like to see this given a different name, since it's a different value (nanosec, not seconds)